code/modules/power/antimatter/control.dm

```diff
@@ -85,16 +88,26 @@
 	if(fuel > (2*core_power))//More fuel has been put in than the current cores can deal with
 		if(prob(50))
 			core_damage = 1//Small chance of damage
+			playsound(src.loc, 'sound/effects/bang.ogg', 25, 1)
 		if((fuel-core_power) > 5)
 			core_damage = 5//Now its really starting to overload the cores
+			playsound(src.loc, 'sound/effects/bang.ogg', 50, 1, 5)
 		if((fuel-core_power) > 10)
 			core_damage = 20//Welp now you did it, they wont stand much of this
+			playsound(src.loc, 'sound/effects/bang.ogg', 200, 1, 10) // 5 of these and it gibs in 7 range

```

Comment:
does 200 even do anything

Reply:
good question! i'll go try it right now!